### PR TITLE
Lee/yaml

### PR DESCRIPTION
**Pull request by VirtDev337**

### CODE REVIEW

**Review by whmcquaid — APPROVED** [review]
:shipit:

**Comment by thatoldplatitude** [comment]
Changes need to be tested before we're able to merge to main

**Comment by thatoldplatitude** [comment]
@VirtDev337 I'll push the spacing & small typos.
But theres a few things missing still that we had skipped over rushing at the end.

We made this fancy new cloudinit to create a user with ssh access from keys we generated. But are we trying to ssh in as them?
Sidenote that we didnt get to go over: Forgot yaml doesnt accept commands. so $(cat ID_FILE) in out cloud-init wont work.
Thats when we can use the heredoc in the webserver.sh

File: cloud-init.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+users:
+  - default
+  - name: rp-user
+    sudo: ALL=(ALL) NOPASSWD:ALL
+    ssh-authorized-keys: $(cat ed25519.pub)
```
Comment (stephenlauck) [inline cloud-init.yaml:5]: This def won't work. Know why?

**Pull request closed without merging**